src/OrchardCore.Modules/OrchardCore.ContentFields/ResourceManagementOptionsConfiguration.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.Extensions.Options;
+using OrchardCore.ResourceManagement;
+
+namespace OrchardCore.ContentFields;
+
+public sealed class ResourceManagementOptionsConfiguration : IConfigureOptions<ResourceManagementOptions>
```

Comment:
I think we define Trumbowyg editor inside the Resources project. It's probably best to add this class there and also change its namespace accordingly. At the same time move the script in the Resources project too.

Reply:
I moved the theme script to the `Resources` module and made it generic so it can be used with other modules. However, the other scripts (media.url and media.tags) are specific to this module and have dependencies that must be included in the HTML to function properly. I also found that these scripts are duplicated between the `OrchardCore.ContentFields` and `OrchardCore.Html` modules. It would be better to abstract the editor functionality maybe in a new module with the related functionality, but it is not the scope of this fix.

Reply:
@mdameer But why did you put them in the ContentFields module? Maybe these two can be moved to the media module since it's media specific.

Also the themes script (which you module per my ask to the resources), it's probably best to place in TheAdmin theme since it depends on a script that is only available by TheAdmin theme.

Reply:
For media scripts, they are already in the ContentFields module, but I just moved them to the `js` folder, regarding the theme script, it is a generic script even if it depends on the admin script for now, I think it is better to abstract the theme mode scripts to make it reusable by the admin or public site rather than putting anything depends on it in the admin theme, we can implement it in a separate change, what do you think?

Reply:
I would agree if it did not depend on admin-head script which is in admin. So I think placing it in admin is the way to go. Unless we want to complicate things and move the admin-head script out of TheAdmin theme as well. So I think for now, we should just put it in TheAdmin which should be sufficient for OC for now.

Reply:
you can review the latest changes, I built an abstracted theme manager and moved it to the themes module, with backward compatibility with the old implementation

Reply:
I am not sure how I feel about all these changes. Now we need to inject two resources instead of one for doing the same thing globally. I think it's an unnecessary addition and should be avoided

Reply:
Nothing changed, we need only 'admin-head' to get the same functionality, also we can now use theme manager on the public site to have the same functionality as the admin

Reply:
But now you inject admin-head and theme-manager (2 different scripts)